Tools/Create-Azure-Sentinel-Solution/createSolution.ps1

```diff
@@ -712,6 +713,46 @@ foreach ($inputFile in $(Get-ChildItem $path)) {
                         # Update Connector Counter
                         $connectorCounter += 1
                     }
+                    elseif ($objectKeyLowercase -eq "savedsearches") {
+                        $isStandardTemplate = $false
+                        $searchData = $json # Assume input is basic array of SavedSearches to start
+                        # Check if SavedSearch input file uses direct structure given by export
+                        if ($searchData -isnot [System.Array] -and $searchData.value) {
```

Comment:
@elforb - Can you give us sample input to understand more about the changes? because I can see in the changes that we are using searchdata as array sometimes and sometimes not. Please provide me the sample inputs to understand more about the changes related to savedsearches

Reply:
The expected input format is either of the following:
- Raw array
- Direct format from export at https://docs.microsoft.com/en-us/rest/api/loganalytics/saved-searches/list-by-workspace
- Standard ARM Template

Reply:
@elforb - Got you. In that case If input is of direct structure given by export template and it value prop exists then we don't need to add it to the the basemaintemplate file?

Reply:
@Tichandr No?? We have to add the resources to the template no matter what format they come in.